src/components/StyledButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from 'expo-router';
+import React from 'react';
+import { Pressable, Text, TouchableOpacity } from 'react-native';
+import styles from './styles';
+
+function StyledButton({ text, file }: { text: string; file: string }) {
+  return (
+    <Link href={file} asChild>
+      <Pressable>
```

Comment:
We don't need `Pressable` and `TouchableOpacity` because these are both essentially "buttons". Also, add the style for the button in a `View` around everything rather than in the `TouchableOpacity`.